src/js/background/background.js

```diff
@@ -22,8 +22,8 @@ browser.runtime.onInstalled.addListener(async (details) => {
 // eslint-disable-next-line no-redeclare, no-unused-vars
 async function getAliasesFromServer(method = "GET", opts=null) {
   const { relayApiSource } = await browser.storage.local.get("relayApiSource");  
-  const apiMakeRelayAddressesURL = `${relayApiSource}/relayaddresses/`;
-  const apiMakeDomainAddressesURL = `${relayApiSource}/domainaddresses/`;
+  const getRelayAddressesUrl = `${relayApiSource}/relayaddresses/`;
+  const getDomainAddressesUrl = `${relayApiSource}/domainaddresses/`;
```

Comment:
Nit, but I'd expect names starting with a verb (`get*`) to be functions.

Reply:
Renamed all of the API variables to follow this pattern: 
- `relayApiUrlRelayAddresses`
- `relayApiUrlRelayAddressId`
- `relayApiUrlDomainAddresses`

Revised in 80402c57eaa99b1b1617691f015a18f1ed6fda02